src/Configuration/src/CloudFoundry/CloudFoundryConfigurationProvider.cs

```diff
@@ -38,7 +38,7 @@ internal static Stream GetMemoryStream(string json)
         return stream;
     }
 
-    internal void AddDiegoVariables(IDictionary<string, string> data)
+    private void AddDiegoVariables(IDictionary<string, string> data)
```

Comment:
This might make life difficult for us down the line, if we want to access it internally. Internal leaves room for us, without exposing public surface.

Reply:
How would you envision using this elsewhere? I've always viewed this config provider as a relatively self-contained building block that shouldn't be referenced directly, basically a shim that translates CF env/config info into .NET constructs for other components to build on via IConfiguration or IOptions

Reply:
IIRC, I've made this private simply because it wasn't used anywhere. We can always make it internal again if there's a need, without introducing a breaking change.